exercises/22_name_raise.rb

```diff
@@ -0,0 +1,26 @@
+class Name
+  def initialize(firstname, lastname)
+    if !firstname || !lastname
```

Comment:
Don't use nagation instead use
`unless firstname || lastname`

Reply:
if we are using unless then it should be "firstname && lastname"